vnext/PropertySheets/React.Cpp.props

```diff
@@ -67,6 +67,10 @@
     <Import Project="$(ConfigurationType)\$(Configuration).props" />
   </ImportGroup>
 
+  <PropertyGroup>
+    <CppStandard Condition="'$(CppStandard)'==''">stdcpp17</CppStandard>
```

Comment:
>CppStandard [](http://example.com/codeflow?start=5&length=11)

I would suggest that we start prefixing all RNW project specific variables with a common prefix such as 'React', 'ReactNative', 'ReactNativeWindows', or 'Rnw'. This way we can avoid any possible conflicts with other variables and make it clear to what they belong too.
See as example the C++/WinRT project - they prefix all their variables with the 'CppWinRT' prefix.